packages/forms-web-app/src/controllers/full-appeal/submit-appeal/new-plans-drawings.js

```diff
@@ -1,89 +1,56 @@
-const {
-  documentTypes: {
-    decisionPlans: { name: documentType },
-  },
-} = require('@pins/common');
+const logger = require('../../../lib/logger');
+const { createOrUpdateAppeal } = require('../../../lib/appeals-api-wrapper');
 const {
   VIEW: {
-    FULL_APPEAL: { NEW_PLANS_DRAWINGS, PLANNING_OBLIGATION_PLANNED },
+    FULL_APPEAL: { PLANS_DRAWINGS, NEW_PLANS_DRAWINGS, PLANNING_OBLIGATION_PLANNED },
   },
 } = require('../../../lib/full-appeal/views');
-const logger = require('../../../lib/logger');
-const { createDocument } = require('../../../lib/documents-api-wrapper');
-const { createOrUpdateAppeal } = require('../../../lib/appeals-api-wrapper');
 const { COMPLETED } = require('../../../services/task-status/task-statuses');
 
 const sectionName = 'appealDocumentsSection';
 const taskName = 'plansDrawings';
 
 const getNewPlansDrawings = (req, res) => {
-  const {
-    session: {
-      appeal: {
-        id: appealId,
-        [sectionName]: {
-          [taskName]: { uploadedFiles },
-        },
-      },
-    },
-  } = req;
+  const { hasPlansDrawings } = req.session.appeal[sectionName][taskName];
   res.render(NEW_PLANS_DRAWINGS, {
-    appealId,
-    uploadedFiles,
```

Comment:
Shouldn't the uploadedFiles be part of the render attributes? Or are they not required?